Makefile

```diff
@@ -47,6 +47,16 @@ container:
     --build-arg REPO_INFO=${REPO_INFO} \
     -t ${IMAGE}:${TAG} .
 
+.PHONY: multi-arch
+multi-arch:
+	docker buildx build \
+    --platform linux/amd64,linux/arm64 \
```

Comment:
Does this result in two different images with the same tag? If so, then how are they supposed to be hosted and pulled?

Reply:
I am not 100% sure how it knows what architecture a container was built for... Here when I pull `ubuntu:latest` it just knows... https://hub.docker.com/_/ubuntu?tab=tags I hope maybe @lyndon160 can provide more insight here?

Reply:
This results in two images and one manifest. The manifest then points to each image. When a docker pull is made, it reads the manifest to pull the image that matches your architecture.

If multi-arch images are not supported then simply do the `buildx` command for arm64 and tag it something like kong:1.5-arm64

Unsure about Bintray, they don't have any documentation on this. Most of the popular registries offer multi-arch -- maybe test it locally first with an helloworld image? Again if this doesn't work, then tag images for each arch.